plugins/pwa.ts

```diff
@@ -27,9 +38,15 @@ export default async () => {
         })
 
         notif.$on('close', () => {
-          window.sessionStorage.clear()
-          window.localStorage.clear()
-          window.location.reload()
+          try {
+            window.sessionStorage.clear()
+            window.localStorage.clear()
+            flushIndexedDb()
```

Comment:
i think we need an await here 
```suggestion
            await flushIndexedDb()
```